azurerm/resource_arm_policy_assignment.go

```diff
@@ -57,6 +58,34 @@ func resourceArmPolicyAssignment() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"location": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				ForceNew:         true,
+				StateFunc:        azure.NormalizeLocation,
+				DiffSuppressFunc: azure.SuppressLocationDiff,
+			},
+
+			"identity_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(policy.None),
+					string(policy.SystemAssigned),
```

Comment:
given this is being split out into a block we can update this to only support `SystemAssigned` as a value for the moment (and we can infer `none` as an option)

Reply:
I kept the `None` value in. IMO it doesn't hurt and is "spec" compliant. But happy to remove it if you like.